make.sh

```diff
@@ -28,23 +30,40 @@ kill_port_80_usage () {
     fi
 }
 
+find_hostfile () {
+    # try windows first, mac second; add future platforms here.
+    # make sure there aren't any conflicts e.g. the windows bash host may also provide an /etc/hosts file
+    for hosts_candidate in "C:/windows/system32/drivers/etc/hosts" "/etc/hosts"; do
+        echo -n "looking for hostfile at "$hosts_candidate
```

Comment:
```suggestion
        echo "Looking for hostfile at ${hosts_candidate}"
```
No newline here will only clutter the output.
Capitalized sentence
Keeping the variable in the echo quotes for consistency